README.md

```diff
@@ -283,6 +283,12 @@ application has stopped.
 Stops the application and then starts it. Returns a `Promise` that will be
 resolved once the application has started again.
 
+#### isRunning()
+
+Returns a `Promise` that will be resolved with the statue of the application.
```

Comment:
It does not return a promise, it returns a boolean, so I think this line can be removed.